pkg/collect/host_kernel_modules.go

```diff
@@ -228,7 +228,7 @@ func (l kernelModulesLoaded) collectBuiltin() (map[string]KernelModuleInfo, erro
 	}
 	kernel := strings.TrimSpace(string(out))
 
-	file, err := os.Open(fmt.Sprintf("/usr/lib/modules/%s/modules.builtin", kernel))
+	file, err := os.Open(fmt.Sprintf("/lib/modules/%s/modules.builtin", kernel))
```

Comment:
is it always going to consistently be this dir, no matter the distro?  Maybe we should have a list of possible locations and iterate through them `if os.IsNotExist`

Reply:
I think it _should be fine_ though.  It seems the convention is `/lib/modules`

Reply:
updated with a test